haystack/components/evaluators/document_map.py

```diff
@@ -10,7 +10,7 @@
 @component
 class DocumentMAPEvaluator:
     """
-    A Mean Average Precision (MAP) evaluator for documents.
+    A Mean Average Precision (MAP) evaluator for documents. For details, please refer to the [resource](https://www.pinecone.io/learn/offline-evaluation/).
```

Comment:
```suggestion
    A Mean Average Precision (MAP) evaluator for documents.
```

I would not put this link in the docstring, but I would put it as a comment at the beginning of the `run` method code to help those who will be working with the code in the future.